packages/http/src/__tests__/http-prism-instance.spec.ts

```diff
@@ -16,6 +16,8 @@ const staticExamplesOas2Path = fixturePath('static-examples.oas2.json');
 const serverValidationOas2Path = fixturePath('server-validation.oas2.json');
 const serverValidationOas3Path = fixturePath('server-validation.oas3.json');
 
+const { version: prismVersion } = require('../../package.json');
```

Comment:
@XVincentX is this ok?

Reply:
This should be ok — I'd just not use require and import it.

```suggestion
import { version as prismVersion } from '../../package.json';
```

This will probably give us some type checking as well